botorch/models/multitask.py

```diff
@@ -108,7 +109,7 @@ def get_task_value_remapping(
         # Create a tensor that maps task values to new task values.
         # The number of tasks should be small, so this should be quite efficient.
         mapper = torch.full(
-            (task_values.max().item() + 1,),
+            (int(task_values.max().item()) + 1,),
```

Comment:
What motivated this change? Is this just to make the type-checker happy?

Reply:
yes :)